packit/cli/copr_build.py

```diff
@@ -83,7 +83,16 @@ def copr_build(
     it defaults to the current working directory.
     """
     api = get_packit_api(config=config, local_project=path_or_url)
-    default_project_name = f"packit-cli-{path_or_url.repo_name}-{path_or_url.ref}"
+    config_project_value = [
+        job.metadata.get("project")
+        for job in api.package_config.jobs
+        if job.type == JobType.copr_build and job.metadata.get("project")
+    ]
+
```

Comment:
I'd probably warn here if there are >2 different projects in the metadata that we'll pick the first one

Reply:
Doesn't we expect to have just one project value in the config?

Reply:
You can have multiple triggers for copr builds (e.g. `pull_request` and `release`) with a different configuration.

Usecase:
- default packit naming for PRs
- custom copr repo for release

Reply:
Thanks for the explanation! I will add the warning